packages/amplify_storage_s3/ios/Classes/FlutterDownloadFileRequest.swift

```diff
@@ -26,15 +26,18 @@ struct FlutterDownloadFileRequest {
         self.options = setOptions(request: request)
     }
     
-    static func isValid(request: Dictionary<String, AnyObject>) -> Bool {
-        var valid: Bool = true;
+    static func validate(request: Dictionary<String, AnyObject>) throws {
+        let validationErrorMessage = "DownloadFile request malformed."
         if !(request["key"] is String) {
-            valid = false
+            throw AmplifyFlutterValidationException(errorDescription: validationErrorMessage,
+                                                    recoverySuggestion: "Ensure that key attribute is present.",
```

Comment:
Looks like we missed using the template strings in this file.

Reply:
fixed.